pkgs/applications/audio/reaper/default.nix

```diff
@@ -14,14 +15,16 @@
 , xdotool
 , which
 
-, jackSupport ? true
+, jackSupport ? stdenv.isLinux
 , jackLibrary
-, pulseaudioSupport ? config.pulseaudio or true
+, pulseaudioSupport ? config.pulseaudio or stdenv.isLinux
 , libpulseaudio
 }:
 
 let
-  url_for_platform = version: arch: "https://www.reaper.fm/files/${lib.versions.major version}.x/reaper${builtins.replaceStrings ["."] [""] version}_linux_${arch}.tar.xz";
+  url_for_platform = version: arch: if stdenv.isDarwin
```

Comment:
Since it's universal, I assume it has `x86_64-darwin` support?

Reply:
Possibly. I don't have hardware to check, but I could yolo add it and hope for the best.

Reply:
It does work, but I struggle with updating the hashes in the file. `update-source-version` really does not like it if two archs have the same hash :(

Reply:
ok, the last update works